attributes/network/14_proof.adoc

```diff
@@ -0,0 +1,17 @@
+= Proof Attribute (#14)
+:cddl: ./cddl/
+
+This attribute allows server to deliver the proof of an object requested.
+The proof itself is a sequence of actions used to construct the state in which the object is embedded within the corresponding Merkle tree.
+
+== Attribute Argument
+
+This is a uint representing the hasher to be used in constructing proofs.
+
+* 0 - merk default hash (see xref:../../spec/proof_hash_scheme.adoc.)
```

Comment:
I would not put any implementation detail here.

We should be able to query the server and retrieve the hashes it supports.

Reply:
@fmorency I believe this contradicts what @hansl just requested.

Reply:
@hansl WDTY?